packages/generate-persisted-query-manifest/README.md

```diff
@@ -96,6 +96,27 @@ Default:
 ]
 ```
 
+#### Usage with GraphQL Codegen persisted documents
+
+[GraphQL Codegen](https://the-guild.dev/graphql/codegen) is a popular code
+generation utility used with GraphQL. You can use GraphQL Codegen's [persisted
+documents](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client#persisted-documents) feature with `generate-persisted-query-manifest` by providing the `documents` option with the `fromGraphQLCodegenPersistedDocuments` utility exported by this package. Doing so will skip file system traversal and instead use the GraphQL documents defined in the persisted documents manifest as the documents source.
+
+```ts
+import type { PersistedQueryManifestConfig } from "@apollo/generate-persisted-query-manifest";
+import { fromGraphQLCodegenPersistedDocuments } from "@apollo/generate-persisted-query-manifest";
+
+const config: PersistedQueryManifestConfig = {
+  documents: fromGraphQLCodegenPersistedDocuments(
+    "./src/gql/persisted-documents.json",
+  ),
+};
+
+export default config;
+```
+
+> NOTE: Running these documents through this package is necessary to ensure all document transforms built into Apollo Client are also run on these documents. This ensures forward compatibility with any future changes introduced by Apollo Client that may alter the GraphQL document output.
```

Comment:
This sort of explains the opposite issue to the one I'm concerned about above (that one could imagine just taking the codegen JSON and generating our manifest from it directly rather than using ApolloClient like this tool does), whereas my thought above is about "can't you just use the normal version of this tool directly on your app source".

Reply:
Funny I think this shows the 2 different ways you and I think about this 🤣. Some of this comes form insider knowledge on whats coming with Apollo Client, but you have a solid point here. I think it might be worth expressing both.

Reply:
Took a stab at this with https://github.com/apollographql/apollo-utils/pull/414/commits/b16fcf2ea89f643821c1a4fdc80d777d5d29c7f4. Let me know what you think!